webapp/loggedin/modules/home/dashboard/surveyInfo/deleteSurveyDialog.js

```diff
@@ -9,70 +12,66 @@ import {
   ModalFooter,
 } from '../../../../../commonComponents/modal'
 
-export default class DeleteSurveyDialog extends React.Component {
-  constructor (props) {
-    super(props)
-
-    this.state = {confirmName: ''}
-    this.confirmNameOnChange = this.confirmNameOnChange.bind(this)
-    this.confirmNameRef = React.createRef()
-  }
+const confirmNameRef = React.createRef()
 
```

Comment:
it's preferred to have the variable in the scope of where it's used. in this case inside the component. Or do you use it outside it?

Reply:
Moved inside function